config/application.rb

```diff
@@ -21,5 +21,7 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.autoload_paths << Rails.root.join('app', 'graph')
+    config.autoload_paths << Rails.root.join('app', 'graph', 'mutations')
```

Comment:
I wonder if this one is needed? It doesn't automagically do subfolders if namespace matches folder name?